cmd/process-agent/main.go

```diff
@@ -5,9 +5,18 @@ package main
 import (
 	"os"
 
+	"github.com/spf13/cobra"
```

Comment:
nit: should this put after the `"github.com/DataDog/datadog-agent/cmd/process-agent/flags"`

Reply:
It seems like some people like to use the python style of grouping std libs, then 3rd party libs, and lastly application libs. No preference either way, but perhaps we should converge on one or the other?
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Indeed there isn't consistency about it, but I do think in areas that feel like better more polished code (to me personally 😄) we seem to follow this order (standard, 3rdparty, project).